docs/examples/driver_examples/QCoDeS example with CopperMountain_M5065.ipynb

```diff
@@ -0,0 +1,537 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "marked-capability",
+   "metadata": {},
+   "source": [
+    "# Example with Copper Mountain Model M5065 Vector Network Analyzer\n",
+    "This notebook was adapted from `qcodes_contrib_drivers` for Copper Mountain M5180. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "placed-maldives",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "NoTagError: `git describe --long --dirty --always --tags '--match=v*'` could not find a tag\n"
```

Comment:
Could we remove this output from the file. I think its because its generated from an editble install where tags are missing